ODPi-Runtime.md

```diff
@@ -273,36 +268,36 @@ Best practices for compatible apps to be portable and operator friendly:
 
 -   Hive allows users to set some configuration values as part of their Hive session, via the *set* command.  Applications SHOULD only depend on setting those values that Hive by default allows users to set.  These values are listed in Appendix B.
 
 Glossary
 ========
 
--   **Service** - A *service* refers to a software package that is installable within the Hadoop stack. A service can be comprised of one or more *components* (such as a NameNode, DataNode, etc.) or may be as simple as a single library.
+-   **Service** - Software package that is installable within the Hadoop stack. A service can be comprised of one or more *components* (such as a NameNode, DataNode, etc.) or may be as simple as a single library.
 
 -   **Component** - A *service* is comprised of one or more components. For example, HDFS has three components: NameNode, Secondary NameNode, and DataNode. A single component may be installed across multiple nodes in the cluster, such as in the case of the HDFS data node.
 
--   **Distribution** - A collection of components.
+-   **Platform** - A product or service offering that consists of Apache Hadoop™, and optionally includes additional open source and/or commercial components that work with Apache Hadoop™.
 
--   **ISV vendor** - Individual or company that created an ISV application.
+-   **Platform Vendor** - An organization, either a commercial vendor or an open source project, that either produces a packaged version of the Platform or supports a service offering around the Platform aimed at End User consumption.
 
--   **ISV application** - Non-ODPi application or process that runs on top of or beside an ODPi platform.
+-   **Application** - A product that is intended to interoperate with a Platform.
 
--   **ODPi Platform** - A distribution of software that includes all of the required components and is compliant with this specification.
+-   **Software Vendor** - An organization, either a commercial vendor or an open source project, that has a product that works with a Platform.
 
--   **ODPi Runtime** - ODPi specification and platforms geared towards holistic management.
+-   **ODPi Interoperable Application** - An Application that has been verified by the Software Vendor and ODPi to be ODPi Interoperable, verified by completeing the compliance process outlined at https://github.com/odpi/self-certification-reports/blob/master/COMPLIANCE_PROCESS.md#odpi-interoperable.
 
--   **ODPi Core** - ODPi specification and platforms geared towards components outside of any management requirements.
+-   **Reference Platform** - A Platform Vendor neutral, open source Plaform that is composed of projects outlined in the *Included Projects* in this document, leverages the binary packages made available by the associated Apache project, and passes all acceptance tests for this version of the ODPi Runtime Specification.
```

Comment:
That's a fair point - I think we could say 'binary packages and/or source code' here. @rvs you agree?